### PR TITLE
Remove link from prereq, use builds directory

### PR DESCRIPTION
**Pull request by sawpresto**
Note: In the default recipe that is used in the HOL, it copies the contents from https://raw.githubusercontent.com/dtzar/chef/master/workshop/mrpapp/recipes/default.rb. If you want the builds directory to be https://github.com/Microsoft/PartsUnlimitedMRP/tree/master/builds instead, you may want to change that reference in the default.rb to be copied (then remove task 4, step 13 in the HOL).

### CODE REVIEW

File: docs/HOL_Provisioning-Deploying-Environments-Using-Chef/HOL_Provisioning-Deploying-Environments-Using-Chef.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ In this hands-on lab you will explore some of the new features and capabilities
 
 - An SSH client such as PuTTY
 
-- PartsUnlimitedMRP deployed to an Azure Linux Virtual Machine (see [https://github.com/Microsoft/PartsUnlimitedMRP/blob/master/docs/Build-MRP-App-Linux.md](https://github.com/Microsoft/PartsUnlimitedMRP/blob/master/docs/Build-MRP-App-Linux.md))
```
Comment (dtzar) [inline docs/HOL_Provisioning-Deploying-Environments-Using-Chef/HOL_Provisioning-Deploying-Environments-Using-Chef.md]: you need to use the raw.githubusercontent... link in order for all of these to work

**Comment by dtzar** [comment]
Also, please fix the link to my person GitHub repo in step 8.  Other than this and the raw.githubusercontent link fix we should be good to go.

File: docs/HOL_Provisioning-Deploying-Environments-Using-Chef/HOL_Provisioning-Deploying-Environments-Using-Chef.md (hunk 1 above)
Comment (sawpresto) [inline docs/HOL_Provisioning-Deploying-Environments-Using-Chef/HOL_Provisioning-Deploying-Environments-Using-Chef.md]: Done!

**Comment by sawpresto** [comment]
Done!

**Pull request merged**